package.json

```diff
@@ -2,6 +2,7 @@
   "name": "flock-eco-workday",
   "main": "index.js",
   "scripts": {
+    "compile": "tsc",
```

Comment:
Can we make this part of the 'build' command instead? I'm not too familiar yet anymore with the nom scripts, but build seems like a sensible one to include this one too

Reply:
Will change this